packages/apm/src/integrations/tracing.ts

```diff
@@ -760,7 +760,9 @@ export class Tracing implements Integration {
       });
 
       Tracing._forceLCP = (): void => {
-        po.takeRecords().forEach(updateLCP);
+        if (po.takeRecords) {
+          po.takeRecords().forEach(updateLCP);
+        }
```

Comment:
Hmm, wait a sec, wasn't the missing method covered by the `try..catch`?

Reply:
You should avoid exceptions as much as possible since they're costly.